apps/events-helsinki/src/domain/search/eventSearch/__tests__/utils.test.ts

```diff
@@ -51,7 +51,7 @@ describe('getEventSearchVariables function', () => {
   const defaultParams = {
     include: [],
     language: 'fi' as AppLanguage,
-    pageSize: 10,
+    pageSize: 25,
```

Comment:
sortOrder comes from a constant. Should this come from one too? The snapshots are comparing the used value anyway.

Reply:
Added refactoring to move pageSize to AppConfig for all apps in df802da46e6936cc07e35534b874000252f8935b